docs/workers.md

```diff
@@ -256,6 +259,7 @@ ensure that the purge history admin API is not used while pagination requests
 for the room are in flight:
 
     ^/_matrix/client/(api/v1|r0|unstable)/rooms/.*/messages$
+    ^/_matrix/client/unstable/org.matrix.msc2946/rooms/.*/hierarchy$
```

Comment:
I think ensuring all requests go to the same worker would be fixed by #10613, but it is unclear if that's the proper approach.